boards/arm/usb_kw24d512/doc/usb_kw24d512.rst

```diff
@@ -0,0 +1,127 @@
++-----------+------------+-------------------------------------+
+| SPI       | on-chip    | spi                                 |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| FLASH     | on-chip    | soc flash                           |
++-----------+------------+-------------------------------------+
+
+The default configuration can be found in the defconfig file:
+
+	``boards/arm/usb_kw24d512/usb_kw24d512_defconfig``
+
+Other hardware features are not currently supported by the port.
+
+Connections and IOs
+===================
+
+The KW2xD SoC has five pairs of pinmux/gpio controllers.
+
++-------+-----------------+--------------------------------------+
+| Name  | Function        | Usage                                |
++=======+=================+======================================+
+| PTA1  | UART0_RX        | UART Console                         |
++-------+-----------------+--------------------------------------+
+| PTA2  | UART0_TX        | UART Console                         |
++-------+-----------------+--------------------------------------+
+| PTC4  | GPIO            | SW1                                  |
++-------+-----------------+--------------------------------------+
+| PTD4  | GPIO            | Blue LED (D2)                        |
++-------+-----------------+--------------------------------------+
+| PTD5  | GPIO            | Blue LED (D3)                        |
++-------+-----------------+--------------------------------------+
+| PTB10 | SPI1_PCS0       | internal connected to MCR20A         |
++-------+-----------------+--------------------------------------+
+| PTB11 | SPI1_SCK        | internal connected to MCR20A         |
++-------+-----------------+--------------------------------------+
+| PTB16 | SPI1_SOUT       | internal connected to MCR20A         |
++-------+-----------------+--------------------------------------+
+| PTB17 | SPI1_SIN        | internal connected to MCR20A         |
++-------+-----------------+--------------------------------------+
+| PTB19 | GPIO            | internal connected to MCR20A (Reset) |
++-------+-----------------+--------------------------------------+
+| PTB3  | GPIO            | internal connected to MCR20A (IRQ_B) |
++-------+-----------------+--------------------------------------+
+| PTC0  | GPIO            | internal connected to MCR20A (GPIO5) |
++-------+-----------------+--------------------------------------+
+
+System Clock
+============
+
+The KW2xD SoC is configured to use the 4 MHz external clock from the modem
+with the on-chip PLL to generate a 48 MHz system clock.
+
+Serial Port
+===========
+
+The KW2xD SoC has six UARTs. One is configured and can be used for the console,
+the remaining are not used.
+
```

Comment:
You're missing the "Programming and Debugging" section listed in the board doc template (doc/templates/board.tmpl). Can you say something about this?


Flashing
========
[How to use this board with Zephyr and how to flash a Zephyr binary on this
device]


Debugging
=========
[ How to debug this board]